src/Writers/HTMLWriter.jl

```diff
@@ -464,6 +505,17 @@ module RD
             """
         ))
     end
+    function mathengine!(r::RequireJS, engine::MathJax3)
+        push!(r, RemoteLibrary(
+            "mathjax3",
+            "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-svg.js",
+        ))
+        push!(r, Snippet(["mathjax3"], ["MathJax"],
+            """
+            window.MathJax = $(json_jsescape(engine.config, 2))
```

Comment:
This doesn't work. I would appreciate any hints from people more familiar with Node and `require` on how to correctly configure this.

Reply:
I think that's expected, since RequireJS loads the library before `require` calls the configuration callback. It looks like they've done away with the `Config()` function, so I couldn't really quickly figure out what would be the correct way of doing it is. But this might be relevant: https://docs.mathjax.org/en/latest/web/configuration.html#configuring-mathjax-after-it-is-loaded

Reply:
Is there a reason, we require the callback to be called after the library is loaded? Couldn't we just configure mathjax before it is loaded?

Reply:
Well, that's just how RequireJS works (afaiu). I don't know if it possible to somehow run some code before RequireJS fetches and runs the module. Maybe it's possible to declare a custom dependency module somehow that RequireJS would have to load before MathJax..?

An alternative would be to drop RequireJS altogether for this, but currently the whole `documenter.js` generation is built around RequireJS..